changelog.d/saf-991.fixed

```diff
@@ -0,0 +1,5 @@
+The SARIF output format should have the tag "security" when the "cwe"
+section is present in the rule. Moreover, duplicate tags should be
+de-duped.
+
+Osemgrep wasn't doing this before, but with this fix, now it does.
```

Comment:
maybe if it concerns only osemgrep, we could add a osemgrep: at the beginning of this changelog instead of at the end.

Reply:
Will do so in the future ones!